src/screens/selectPlayers/index.tsx

```diff
@@ -80,7 +90,7 @@ const SelectPlayers = ({
         <Text style={styles.wideRowText}>
           {item?.firstName} {item?.lastName}
         </Text>
-        <Text style={styles.rowText}>{item.fidelity}</Text>
+        <Text style={styles.rowText}>{skillChecker(item.skill)}</Text>
```

Comment:
Why using a `<Text>` here if `skillChecker()` is returning an IconComponent? A plain `View` is more adequate